src/Matcher.php

```diff
@@ -131,20 +135,18 @@ public function match(ServerRequestInterface $request)
      *
      * @param Route $proto The proto-route to match against.
      *
-     * @param string $name The route name.
-     *
      * @param string $path The request path.
      *
      * @return mixed False on failure, or a Route on match.
      *
      */
-    protected function requestRoute($request, $proto, $name, $path)
+    protected function requestRoute($request, $proto, $path)
     {
         if (! $proto->isRoutable) {
-            return;
+            return false;
```

Comment:
See doctype description: `@return mixed False on failure, or a Route on match.`
so this should return false?